@here/harp-mapview/test/PolarTileDataSourceTest.ts

```diff
@@ -238,7 +238,7 @@ describe("PolarTileDataSource", function() {
             assert.equal(tile2.objects.length, 0);
         });
 
-        it("Geometry should not exceed pole radius", function() {
+        it.skip("Geometry should not exceed pole radius", function() {
```

Comment:
This test seem to fail on windows/chrome

Reply:
I'll check on windows.